README.md

```diff
@@ -1,7 +1,331 @@
 # Coveo C# Platform SDK
-The offical Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+[![NuGet version (Coveo.Connectors.Utilities.PlatformSdk)](https://img.shields.io/nuget/v/Coveo.Connectors.Utilities.PlatformSdk.svg?style=flat-square)](https://www.nuget.org/packages/Coveo.Connectors.Utilities.PlatformSdk/)
 
-The code of the SDK is not open-source. This repo is meant to provide feedback, bug reporting, etc.
+The official Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+
+The package is .NET Standard 2.0 compatible and therefore works with .NET Core and .NET Framework
+
+The code of the SDK is not open-source. This repository is meant to provide feedback, bug reporting, etc.
+
+## Installation
+Install the public NuGet package [Coveo.Connectors.Utilities.PlatformSdk](https://www.nuget.org/packages/Coveo.Connectors.Utilities.PlatformSdk/) in your C# project using your favorite IDE or NuGet command line.
+
+The package is signed before being published to NuGet to avoid pulling a malicious package.
+We provide the `PDBs` to have a better debugging experience.
+Documentation is available for most of public members and methods.
 
 ## How to use the SDK
-Coming soon!
+First, you need to instantiate the client to interact with the platform. Here is the minimum configuration you need to provide:
+
+
+```csharp
+string apiKey = "Your api key with required rights";
```

Comment:
Is it clear somewhere how to get an api key with required rights? Would you be worth adding a comment above to the Coveo Online Help about it like : https://docs.coveo.com/en/82/cloud-v2-developers/creating-an-api-key ?

Reply:
Or the Api Key section of this page https://docs.coveo.com/en/1546/cloud-v2-administrators/add-or-edit-a-push-source#add-or-edit-a-push-source-1 which basically lets you check a box and automatically create a key with the correct privileges.

Reply:
Voir commentaire plus haut et celui que j'ai rajouté dans la section Push Api document